build.gradle

```diff
@@ -499,3 +504,27 @@ task testExampleService(type: com.marklogic.gradle.task.MarkLogicTask) {
   }
 }
 
+task configureUnauthorizedKoop {
```

Comment:
Please add a comment indicating this is for testing.

Reply:
done